app/templates/components/event-card.hbs

```diff
@@ -23,10 +23,18 @@
       </SmartOverflow>
       <div class="meta">
         <span class="date">
-          {{general-date this.event.startsAt tz=this.event.timezone}}
```

Comment:
Why do you think we had `tz=this.event.timezone`. We had not written it for fun, there was a reason for it. Why did you remove it?